### DISCUSSION TITLE
How to set the default mode to dark?

### DISCUSSION THREAD

**Discussion by enriquedlh97** [themesberg/flowbite-react#875] [Q&A] [answered]
How can I make sure that the starting theme mode is dark and not light?

**Comment by tulup-conner** [comment]
You can accomplish this by wrapping your app in the following:

```js
<Flowbite theme={{ dark: true }}>
 ..
</Flowbite>
```

**Reply by codenomnom** [reply]
Unfortunately - **outdated**.

**Comment by codenomnom** [comment, marked as answer]
As of today, [the way](https://github.com/themesberg/flowbite-react/blob/main/src/components/Flowbite/Flowbite.tsx#L7) to do it is by setting the `mode` property of `Flowbite`:

```js
<Flowbite theme={{ mode: 'dark' }}>
  ...
</Flowbite>
```
